tasks/selinux_centos.yml

```diff
@@ -0,0 +1,10 @@
+- name: Set http_can_network_connect
+  seboolean:
+    name: httpd_can_network_connect
+    state: yes
+    persistent: yes
+
+# Disable SELinux
+- name: Disable SELinux (shouldn't be done!)
+  selinux:
+    state: disabled
```

Comment:
It complained about nginx log files. They are type `usr_t` and when i allow nginx access to this type i get warning it is a generic type. I suppose many files are of this type so allowing access is almost as disabling SELinux.